specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/computeRPCommon.json

```diff
@@ -1798,9 +1802,49 @@
             "name": "LinuxPatchAssessmentMode",
             "modelAsString": true
           }
+        },
+        "automaticByPlatformSettings": {
+          "$ref": "#/definitions/LinuxAutomaticByPatchSettings",
+          "description": "Specifies additional settings for patch mode AutomaticByPlatform in VM Guest Patching on Linux."
         }
       },
       "description": "Specifies settings related to VM Guest Patching on Linux."
+    },
+    "WindowsAutomaticByPatchSettings": {
+      "properties": {
+        "rebootSetting": {
+          "type": "string",
+          "description": "Specifies reboot setting for all AutomaticByPlatform patch installation operations.",
+          "enum": [
+            "Unknown",
```

Comment:
@kjohn-msft @mirichmo Please validate if Unknown should be included in this. I added it here because it is in the entity model: 
![image](https://user-images.githubusercontent.com/56841542/162053909-8428c638-8ed1-4e02-9755-9478dd1e7ab8.png)

Not adding a link to the repo here, but for reference, this is in CoreEntityModel-VMGuestPatch.cs

Reply:
The general pattern appears to be matching the underlying enum. In this case keeping Unknown is appropriate.

Reply:
Agreed